Doc/library/ssl.rst

```diff
@@ -2669,7 +2661,8 @@ to specify :const:`CERT_REQUIRED` and similarly check the client certificate.
 Protocol versions
 '''''''''''''''''
 
-SSL versions 2 and 3 are considered insecure and are therefore dangerous to
+SSL versions 2 and 3, TLS 1.0, and TLS 1.1 are considered insecure and are
+therefore dangerous to
 use.  If you want maximum compatibility between clients and servers, it is
```

Comment:
If https://datatracker.ietf.org/doc/rfc8996/ is authoritative, maybe adding it as a source would be useful?